lib/node_modules/@stdlib/blas/base/srotg/docs/types/test.ts

```diff
@@ -0,0 +1,112 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/// <reference types="@stdlib/types"/>
+
+import srotg = require( './index' );
+
+
+// TESTS //
+
+// The function returns a collection...
+{
+	srotg( 0.0, 2.0 ); // $ExpectType Collection
+}
+
+// The compiler throws an error if the function is provided an argument which is not a number...
+{
+	srotg( true, false ); // $ExpectError
```

Comment:
Typically, convention is to hold one param constant while supplying wrong values for the other param and then repeating for the other param, where the first param is constant (and valid) and the second param is supplied various invalid arg types.

Reply:
This is still unresolved. Are you planning on updating?

Reply:
Yeah I will do that.

Reply:
@Pranavchiku This still needs to be updated.

Reply:
Updated this.